### DISCUSSION TITLE
Impianto automazione domotico

### DISCUSSION THREAD

**Discussion by mloris** [indomus/forum#3059] [GENERALE - Discussioni generiche]
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/LORIS~1.MIC/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

<link rel=themeData
href="file:///C:/Users/LORIS~1.MIC/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/LORIS~1.MIC/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=IT style='tab-interval:35.4pt'>

<p class=MsoNormal>Buongiorno,<o:p></o:p></p>

<p class=MsoNormal>vorrei realizzare un sistema domotico controllabile anche da
remoto tramite smartphone.<o:p></o:p></p>

<p class=MsoNormal>Pensavo a Home Assistant su Rasperry da integrare…<o:p></o:p></p>

<p class=MsoNormal>Spiego brevemente quanto vorrei realizzare:<o:p></o:p></p>

<p class=MsoNormal>Vorrei gestire/controllare le seguenti informazioni che ho
suddiviso per tipo di collegamento :<o:p></o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

CONNESSIONE WIFI | CONNESSIONE CAVO
-- | --
5 Termostati Ambiente   ( ero orientato su Meross Termostato WiFi ) | Misura Temperatura parte alta Puffer accumulo
Sensori wifi a batteria su infissi    (per disabilitare climatizzazione quando il contatto è   aperto) | Misura Temperatura parte alta Puffer accumulo
Telecamera esterne Nord | Misura Temperatura mandata riscaldamento
Telecamera esterne Ovest | Misura Temperatura Ritorno riscaldamento
  | Misura Temperatura esterna
  | Misura Watt assorbimento PDC
  | Misura Watt immissione in rete
  | Comando apertura cancello
  | Comando accensione luci
  | Comando a disposizione
  | Comando a disposizione

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal>Mi date qualche dritta da dove iniziare o qualche esempio da
“copiare”<span style='mso-spacerun:yes'>  </span>??<o:p></o:p></p>

<p class=MsoNormal>Vi ringrazio<o:p></o:p></p>

</body>

</html>

**Comment by KC-inDomus** [comment]
puoi fare sostanzialmente tutto. di base però ti consigliamo di usare mini pc e non raspberry, tecnologia ormai superarata.
parti da qui: https://indomus.it/formazione/home-assistant-su-mini-pc-intel-nuc-le-varie-installazioni-le-cose-da-fare/

**Comment by mloris** [comment]
grazie ma che assorbimeno elettrico hanno i miniPC rispetto ai rasberry?
Avendo potenza di calcolo maggiore penso che anche l'assrbimento elettrico sia superiore...

**Reply by Bergoglio** [reply]
La differenza è minima e vale ampiamente la pena.
Incidono sui consumi decisamente di più un forno elettrico, una lavatrice, una asciugatrice......

**Reply by 50494554524F** [reply]
posso darti una misura precisa... 

mb asrock n100dc+32g ram+ssd+hdd 2,5''+dongle wifi+dongle bt in idle consuma 14-14,5w con alimentatore notebook da 90w (questo avevo già, con uno da 60w potresti tagliare qualcosina)

edit: scordato... non ho ancora provato ottimizzazioni, i consumi sono per haos che gira in vm

rpi4 microsd+hdd 2,5''+dongle bt in idle consuma 7,5w con alimentatore originale

rpi5 consuma un po' meno dell'n100, solo per il prezzo

**Comment by KC-inDomus** [comment]
> grazie ma che assorbimeno elettrico hanno i miniPC rispetto ai rasberry? Avendo potenza di calcolo maggiore penso che anche l'assrbimento elettrico sia superiore...

rpi5 non regge il confronto su nessun fronte, rispetto ai mini pc, per l'uso tipico di server domestico per la domotica, non solo sul tema consumi.

**Reply by mloris** [reply]
bene, facciamo 2 calcoli allora
ore annue accensione 24x365= 8760  - costo 0.30€ kW
rpi5 consuma 19.7 €
mini pc consuma= 40 €

**Reply by KC-inDomus** [reply]
LOL, ma questi numeri al lotto? :) i consumi come li hai calcolati?

**Reply by mloris** [reply]
rpi5 = (7.5 W) 0.0075 kW x 8760 ore annue x 0.30€ kW = 19.7€
miniPC = (15 W) 0.015 kW x 8760 ore annue x 0.30€ kW = 39.42€

**Reply by KC-inDomus** [reply]
eh, ma non assorbono così 😂 innanzitutto devi indicare lo stack e il carico cpu, per avere paragoni sensati - e parità di stack "classico" x domotica, ti posso garantire che un N100 consuma meno di un rpi5.
